scripts/github-actions/tests-e2e-direct.sh

```diff
@@ -24,7 +24,7 @@ cd ${REPO_ROOT}/
 echo "Downloading envtest assets..."
 export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
 
-export KCC_USE_DIRECT_RECONCILERS=MonitoringDashboard
+KCC_USE_DIRECT_RECONCILERS=MonitoringDashboard
```

Comment:
I think we need export, otherwise it doesn't apply to subsequent requests, but as this is passing tests let's go with it!

Reply:
I fix this in Part 2. Want to use this as an example to highlight some changes.